mm2src/coins/utxo/utxo_common.rs

```diff
@@ -230,7 +230,7 @@ where
 
 /// Requests balances of the given `addresses`.
 /// The pairs `(Address, CoinBalance)` are guaranteed to be in the same order in which they were requested.
```

Comment:
the doc comment should mention `CoinBalanceMap` instead.

tbh though, i feel we shouldn't make it a map in this low level in the first place. i think converting this to a map for the rpc fits better in callers above.
someone reading just the signature of the method won't make much sense of it.

Reply:
> tbh though, i feel we shouldn't make it a map in this low level in the first place. i think converting this to a map for the rpc fits better in callers above.
someone reading just the signature of the method won't make much sense of it.

Agreed. I just wanted to avoid the extra overhead of an additional iterator. Will work on making this better next week.

Reply:
Fixed